src/node_modules/menu/index.js

```diff
@@ -54,18 +56,18 @@ const template = [
           dialog.showMessageBox(opts)
         }
       },
-      {type: 'separator'},
-      {role: 'resetzoom'},
-      {role: 'zoomin'},
-      {role: 'zoomout'},
-      {type: 'separator'},
-      {role: 'togglefullscreen'}
+      { type: 'separator' },
+      { role: 'resetzoom' },
+      { role: 'zoomin' },
+      { role: 'zoomout' },
+      { type: 'separator' },
+      { role: 'togglefullscreen' }
```

Comment:
+1 on the spaces on either side of the menu options, I just grabbed these from the electron docs and they were like that already.